.github/workflows/filtered_github_webhooks.yml

```diff
@@ -0,0 +1,33 @@
+# Filters GitHub webhooks to avoid noise in the Discord server.
+name: Execute Filtered GitHub Webhook
+
+on:
+  pull_request:
+    types:
+      - opened
+      - closed
+      - reopened
+
+jobs:
+  notify_if_not_dependabot:
+    if: ${{ !contains(github.event.pull_request.user.login, 'dependabot') }}
+    runs-on: ubuntu-latest
+    steps:
+    # Discord Webhooks have a maximum size of 4k characters. Truncate the body to avoid a failed API call.
```

Comment:
might be cool to build this into the discord-webhook action itself if 4k is always gonna be the limit, to prevent users from having to include this other action beforehand

Reply:
You're not the first person to request this. I've been on the fence about it because I want to avoid codifying Discord's limit into the action. 

That being said:
- People seem to want it
- The limit hasn't changed in at least 2 years
- The error you get back is awesome, the message is `{"embeds":["0"]}`, so it's not like a user can easily figure out that they've exceeded the limit

Yeah, I'll implement this ASAP and update the PR with the new version.